Modules/ITS/src/ITSFhrTask.cxx

```diff
@@ -657,7 +671,7 @@ void ITSFhrTask::monitorData(o2::framework::ProcessingContext& ctx)
           }
         }
       }
-      mGeneralOccupancy->SetBinContent(istave + 1 + StaveBoundary[mLayer], *(std::max_element(mOccupancy[istave], mOccupancy[istave] + nHicPerStave[lay])));
+      mGeneralOccupancy->SetBinContent(istave + 1 + StaveBoundary[mLayer], *(std::max_element(mOccupancyLane[istave], mOccupancyLane[istave] + nHicPerStave[lay] / 2)));
```

Comment:
I think you should remove "/2" at the end and put "*2" instead in order to find the maximum among all lanes of the stave. Is it correct?

Reply:
confirmed